recipe/migrations/azure_core_cpp1120.yaml

```diff
@@ -0,0 +1,9 @@
+migrator_ts: 1718824596
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+  commit_message: Pin azure-core-cpp 1.12.0
+
+azure_core_cpp:
+  - 1.12.0
```

Comment:
Let's add all the pins here that you had in https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6048, and make sure that they're the most recent version per feedstock.

Reply:
Done in https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6056/commits/6ffa874d6747c6c80c439128ef92e79c6019bddc